libraries/Make.py

```diff
@@ -1,9 +1,14 @@
-import Project, os
+import Project
+import os
+import re
 
 
 def project(target, configuration, directory, needy):
-    if os.path.isfile(os.path.join(directory, 'Makefile')):
+    makefile_path = MakeProject.get_makefile_path(directory)
```

Comment:
i imagined we'd want a bit more magic here since it's not uncommon to see other names like this or "makefile.linux". so this method is definitely a step towards that. lgtm